docs/src/tutorials/large_systems.md

```diff
@@ -137,11 +137,14 @@ Symbolic Sparsity Detection. See the manual entry on
 using BenchmarkTools # for @btime
 
 @btime solve(prob_brusselator_2d, NewtonRaphson());
-@btime solve(prob_brusselator_2d, NewtonRaphson(; autodiff = AutoSparseForwardDiff()));
 @btime solve(prob_brusselator_2d,
-    NewtonRaphson(; autodiff = AutoSparseForwardDiff(), linsolve = KLUFactorization()));
+    NewtonRaphson(; autodiff = AutoSparseForwardDiff(; chunksize = 32)));
 @btime solve(prob_brusselator_2d,
-    NewtonRaphson(; autodiff = AutoSparseForwardDiff(), linsolve = KrylovJL_GMRES()));
+    NewtonRaphson(; autodiff = AutoSparseForwardDiff(; chunksize = 32),
+        linsolve = KLUFactorization()));
+@btime solve(prob_brusselator_2d,
+    NewtonRaphson(; autodiff = AutoSparseForwardDiff(; chunksize = 32),
+        linsolve = KrylovJL_GMRES()));
```

Comment:
That's a huge chunk size. Is it actually more helpful than 7?

Reply:
In this case it actually gave a speedup locally till 32 but larger than this made the code slow.